doc/zephyr.doxyfile

```diff
@@ -830,7 +830,7 @@ EXCLUDE_PATTERNS       =
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories use the pattern */test/*
 
-EXCLUDE_SYMBOLS        =
+EXCLUDE_SYMBOLS        = _*
```

Comment:
Looks like this caused a disruption in the force (and caused a different warning message about unnamed nested unions).  Let's change this to:

    # Hide internal names (starting with an underscore, and doxygen-generated names
    # for nested unnamed unions that don't generate meaningful sphinx output anyway.
    EXCLUDE_SYMBOLS        = _*, *.__unnamed__

I tested this building the docs locally and it removed nearly all the warnings caused by the nested unnamed unions.

Reply:
It seems to work locally here too.  I've updated the PR with the suggested change.